Yoast/javascript
# Add Italian morphology support

## Summary


This PR can be summarized in the following changelog entry:

* [yoastseo] Adds a stemmer for the Italian language.
* [Yoast SEO Premium] Adds word form support for Italian.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* In Italian Premium: use an Italian keyphrase such as e.g. `imparare` and use different forms of the keyphrase and in text/title/meta description and check that everything gets recognized. E.g., `imparo`, `imparano`, `imparavo`, `imparavi`, `imparò`, `imparerei`, `imparato`.

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes

## Quality assurance


Fixes #
